.gitignore

```diff
@@ -39,6 +39,8 @@ htdocs/includes/phpunit/
 htdocs/includes/php-parallel-lint/
 htdocs/includes/sebastian/
 htdocs/includes/squizlabs/
+htdocs/includes/stripe/
```

Comment:
We must exclude here only the directory that are not necessary to have dolibarr working.
All package and lib that are required for Dolibarr used must be included to be able to have CI working and packages than can't use composer complete (for example doliwamp.exe, zip distribution or debian or rpm packages). 
Also installations using git clone as deployment tool clone must be immediatly working.